packages/ts/react-crud/src/autoform-field.tsx

```diff
@@ -25,62 +31,62 @@ function getPropertyModel(form: UseFormResult<any>, propertyInfo: PropertyInfo)
 
 type AutoFormTextFieldProps = SharedFieldProps & TextFieldProps;
 
-function AutoFormTextField({ propertyInfo, form, ...other }: AutoFormTextFieldProps) {
+function AutoFormTextField({ propertyInfo, form, options, label, ...other }: AutoFormTextFieldProps) {
```

Comment:
`options` is not mentioned below, was it intentional?

Reply:
Yes, at this point we are not using any of the options in those set of functions, so adding it would result in `unused` ES Lint errors.